src/Features/Core/Portable/Completion/Providers/AbstractDocCommentCompletionProvider.cs

```diff
@@ -279,6 +283,13 @@ public override async Task<CompletionChange> GetChangeAsync(Document document, C
         var replacementText = beforeCaretText;
         var newPosition = replacementSpan.Start + beforeCaretText.Length;
 
+        if (text.Length > replacementSpan.End + 1
+            && text[replacementSpan.End] == '='
+            && text[replacementSpan.End + 1] == '"')
+        {
+            newPosition += 2;
```

Comment:
so this means that we shoudl put the caret between teh quotes, even if there were already quotes there, right?  In that case, can you add that validation in the tests as well, so we don't regress that?  Thanks!